Sources/SwiftDocC/Utility/Synchronization.swift

```diff
@@ -53,6 +56,10 @@ public class Synchronized<Value> {
         lock = UnsafeMutablePointer<pthread_mutex_t>.allocate(capacity: 1)
         lock.initialize(to: pthread_mutex_t())
         pthread_mutex_init(lock, nil)
+        #elseif os(FreeBSD)
+        lock = UnsafeMutablePointer<pthread_mutex_t?>.allocate(capacity: 1)
+        lock.initialize(to: nil)
```

Comment:
Should this be an initialized to `pthread_mutex_t()` like above (or rolled into the same `#if` scopes as the above?